src/targets/crates.ts

```diff
@@ -202,9 +209,23 @@ export class CratesTarget extends BaseTarget {
       '--manifest-path',
       crate.manifest_path,
     ];
-    return spawnProcess(CARGO_BIN, args, {
-      env: { ...process.env, CARGO_REGISTRY_TOKEN: this.cratesConfig.apiToken },
-    });
+    const env = {
+      ...process.env,
+      CARGO_REGISTRY_TOKEN: this.cratesConfig.apiToken,
+    };
+
+    for (let i = 0; i <= 1; i++) {
+      try {
+        return await spawnProcess(CARGO_BIN, args, { env });
+      } catch (e) {
+        if (i == 0 && e.message.includes(VERSION_ERROR)) {
```

Comment:
```suggestion
        if (i === 0 && e.message.includes(VERSION_ERROR)) {
```
Should fix the linting error.

Reply:
Good point, thanks!